frontend/src/pages/multisig/tx/PageMultisigInfo.js

```diff
@@ -42,8 +48,14 @@ export default function PageMultisigInfo() {
   const wallet = useSelector((state) => state.wallet);
   const networks = useSelector((state) => state.wallet.networks);
   const nameToChainIDs = useSelector((state) => state.wallet.nameToChainIDs);
-
+  const verifyAccountRes = useSelector(
+    (state) => state.multisig.verifyAccountRes
+  );
+  const walletAddress =
+    networks[nameToChainIDs[networkName]]?.walletInfo?.bech32Address;
   const chainId = nameToChainIDs[networkName];
+  const pubkeys = multisigAccountDetails.pubkeys;
```

Comment:
You are expected to provide pubkeys as an array. If it's not an array, an error is thrown.